core/modules/layout/plugins/renderers/layout_renderer_flexible.inc

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * @file
+ * Class file to control the main Layout editor.
+ */
+
+class LayoutRendererFlexible extends LayoutRendererEditor {
+
+  /**
+   * Include rendered HTML prior to the layout.
+   *
+   * @var array
+   */
+  var $region_buttons = array();
+
+  /**
+   * {@inheritdoc}
+   */
+  function __construct(Layout $layout, array $default_template_info) {
+    $this->plugin = $default_template_info['plugin'];
+    $this->admin = TRUE;
+    $this->layout = &$layout;
+    $this->layout_info = $default_template_info;
+
+    // Set the page title region if not already.
+    if (!isset($this->layout_info['regions']['title'])) {
+      $this->layout_info['regions']['title'] = t('Page title');
+    }
+    $this->admin = TRUE;
+
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  function renderRegion($region_id, $blocks) {
+    $output = '';
+    if ($region_id == 'title') {
+      return '';
+    }
+    else {
```

Comment:
Don't need an "else" here since it'll skip anyway with the return just above this.

Generally it's good to avoid using else and ifelse to help flatten the logic. The first "if"  is like a gate. If we pass it we're good to go on.

Reply:
Thanks, this was a modification: everything after the else is actually unnecessary, I just havent cleaned up yet.